packages/api/src/platforms/vtex/clients/commerce/index.ts

```diff
@@ -125,74 +125,45 @@ export const VtexCommerce = (
           deliveryMode?: DeliveryMode | null
           body: ShippingDataBody
         },
-        incrementedAddress?: IncrementedAddress
+        incrementedAddress?: IncrementedAddress,
+        hasDeliveryWindow?: boolean
```

Comment:
I think this would more accurately represent the behavior in which this param is used, what do you think? It'd have to be changed in line 168 as well!

```suggestion
        setDeliveryWindow?: boolean
```